.gitlab-ci.yml

```diff
@@ -62,6 +62,7 @@ run tests:
     - echo $(ls -d ./dist/*.whl | grep $CI_COMMIT_SHORT_SHA)
     - pip3 install -U $(ls -d ./dist/*.whl | grep $CI_COMMIT_SHORT_SHA)
     - make test
+    - scripts/validate-metadata.sh
```

Comment:
We can remove this step from the `publish` stage, no need to do it twice.

Reply:
I am not sure. In the testing stage it doensn't check that the python package version is the same as the git tag (and that should be the correct behaviour). Only in the publish stage it enforces that extra check.

Reply:
Sorry for the delay.
The `CI_COMMIT_TAG ` is exposed through all the jobs only when it's a tag. The version to tag check will happen in the `test` stage.
No harm in keeping it there though.